vast/mediafile.go

```diff
@@ -49,23 +51,20 @@ func (mediaFile *MediaFile) Validate() error {
 		errors = append(errors, ErrMediaFileMissUri)
 	}
 
-	if mediaFile.Bitrate != 0 {
-		if mediaFile.Bitrate < defaults.MIN_VIDEO_BITRATE {
+	if mediaFile.Bitrate != nil {
+		if *mediaFile.Bitrate < defaults.MIN_VIDEO_BITRATE {
 			errors = append(errors, ErrMediaFileBitrateTooLow)
 		}
-
-		if mediaFile.Bitrate > defaults.MAX_VIDEO_BITRATE {
-			errors = append(errors, ErrMediaFileBitrateTooHigh)
-		}
-	} else {
-		if mediaFile.MaxBitrate < defaults.MIN_VIDEO_BITRATE {
-			errors = append(errors, ErrMediaFileBitrateTooLow)
-		}
-
-		if mediaFile.MinBitrate > defaults.MAX_VIDEO_BITRATE {
+		if *mediaFile.Bitrate > defaults.MAX_VIDEO_BITRATE {
 			errors = append(errors, ErrMediaFileBitrateTooHigh)
 		}
 	}
+	if mediaFile.MaxBitrate != nil && *mediaFile.MaxBitrate < defaults.MIN_VIDEO_BITRATE {
```

Comment:
Can you be consistent in the code structure? The above is structured as nested if, here it's padded. Please pick one. Furthermore, I think it will read much nicer, if we can create a helper function to validate the bitrate values.

Reply:
The above is nested because there are two cases to check where as the ones below are both single cases. I wouldn't want to duplicate a nil check.

Reply:
I think with these kinds of validations creating helper functions just adds more lines of code thats not really necessary. We're already going down levels into each `Validate` function to check fields. We should just keep it consistent where we do all the `if` checks under each `Validate` function and if it is an object we call `Validate`.